test/unit/model/v2/test_platforms_section.py

```diff
@@ -104,7 +104,17 @@ def _model_platforms_docker_section_data():
             'purge_networks':
             True,
             'foo':
-            'bar'
+            'bar',
+            'docker_host':
+            'tcp://localhost:12376',
+            'cacert_path':
+            '/foo/bar/ca.pem',
+            'cert_path':
+            '/foo/bar/cert.pem',
+            'key_path':
+            '/foo/bar/key.pem',
+            'tls_verify':
+            'true'
```

Comment:
It should be boolean `True`, not a string `'true'`
```suggestion
            True
```
**Please just click a button in this suggestion, this will apply the change right from the GitHub UI**